{
	"id": "CVE-2007-1137",
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2007-1137",
				"description": "putmail.py in Putmail before 1.4 does not detect when a user attempts to use TLS with a server that does not support it, which causes putmail.py to send the username and password in plaintext while the user believes encryption is in use, and allows remote attackers to obtain sensitive information.",
				"references": [
					{
						"source": "CVE",
						"url": "http://osvdb.org/33764"
					},
					{
						"source": "CVE",
						"url": "http://putmail.sourceforge.net/home.html"
					},
					{
						"source": "CVE",
						"url": "http://secunia.com/advisories/24266"
					},
					{
						"source": "CVE",
						"url": "http://www.securityfocus.com/bid/22718"
					},
					{
						"source": "CVE",
						"url": "http://www.vupen.com/english/advisories/2007/0753"
					},
					{
						"source": "CVE",
						"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/32689"
					},
					{
						"source": "mitre",
						"url": "http://osvdb.org/33764"
					},
					{
						"source": "mitre",
						"url": "http://putmail.sourceforge.net/home.html"
					},
					{
						"source": "mitre",
						"url": "http://secunia.com/advisories/24266"
					},
					{
						"source": "mitre",
						"url": "http://www.securityfocus.com/bid/22718"
					},
					{
						"source": "mitre",
						"url": "http://www.vupen.com/english/advisories/2007/0753"
					},
					{
						"source": "mitre",
						"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/32689"
					}
				],
				"modified": "2024-08-07T12:43:22.608Z"
			}
		}
	],
	"data_source": "mitre-v5"
}
